### PR TITLE
Feature/dsg 1416/events subscriber

### PR DESCRIPTION
**Pull request by jcsuarezm**
## Description

[___JIRA_TICKET_TITLE___](//jira_url)

## Types of changes

Patch

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation change (documentation only, will not require redeployment)
- [ ] Refactor or cleanup (functionality unchanged)
  Minor
- [ ] New feature (non-breaking change which adds functionality)
  Major
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have performed a self-review of my own code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

- [ ] I have confirmed all acceptance criteria defined by the story have been met.

### CODE REVIEW

File: helm/document-management/values.demo.yaml [hunk 1]
```diff
@@ -95,6 +95,7 @@ env:
   GOOGLE_STORAGE_EMULATED: false
   PUBSUB_EMULATOR_HOST: false # set actual "host:port" to enable or false to disable
   PUBSUB_ENABLED: true
```
Comment (jgnuvalence) [inline helm/document-management/values.demo.yaml:97]: I think this property is no longer necessary

File: helm/document-management/values.dev.yaml [hunk 2]
```diff
@@ -95,6 +95,7 @@ env:
   GOOGLE_STORAGE_EMULATED: false
   PUBSUB_EMULATOR_HOST: false # set actual "host:port" to enable or false to disable
   PUBSUB_ENABLED: true
```
Comment (jgnuvalence) [inline helm/document-management/values.dev.yaml:97]: I think this property is no longer necessary

File: helm/document-management/values.local.yaml [hunk 3]
```diff
@@ -120,6 +120,7 @@ env:
   GOOGLE_STORAGE_EMULATED: true
   PUBSUB_EMULATOR_HOST: "pubsub-emulator-service:8085" # set actual "host:port" to enable or false to disable
   PUBSUB_ENABLED: true
```
Comment (jgnuvalence) [inline helm/document-management/values.local.yaml:122]: I think this property is no longer necessary

File: service/src/main/resources/application.yaml [hunk 4]
```diff
@@ -88,6 +89,13 @@ auth:
       private-key: ${SELF_SIGN_PRIVATE_KEY}
       roles: "um:reader, um:application-client"
 
+events:
+  subscriber:
+    subscriptions:
+      - ${EVENTS_DOCUMENT_PROCESSING_SUBSCRIPTION}
```
Comment (jgnuvalence) [inline service/src/main/resources/application.yaml:95]: this should come from environment variables

File: service/src/main/resources/application.yaml [hunk 5]
```diff
@@ -88,6 +89,13 @@ auth:
...
+    topics:
+      - ${EVENTS_DOCUMENT_PROCESSING_TOPIC}
```
Comment (jgnuvalence) [inline service/src/main/resources/application.yaml:97]: this should come from environment variables

**Review by jgnuvalence — CHANGES_REQUESTED** [review]

**Review by jgnuvalence — APPROVED** [review]

File: helm/document-management/values.demo.yaml (hunk 1 above)
Comment (jcsuarezm) [reply helm/document-management/values.demo.yaml]: Thank you Julián, this has been removed.

File: helm/document-management/values.dev.yaml (hunk 2 above)
Comment (jcsuarezm) [reply helm/document-management/values.dev.yaml]: Thank you Julián, this has been removed.

File: helm/document-management/values.local.yaml (hunk 3 above)
Comment (jcsuarezm) [reply helm/document-management/values.local.yaml]: Thank you Julián, this has been removed.

File: service/src/main/resources/application.yaml (hunk 4 above)
Comment (jcsuarezm) [reply service/src/main/resources/application.yaml]: Thank you Julián, this works for multiple topics, but to maintain uniformity I have adopted the pattern used in other services.

File: service/src/main/resources/application.yaml (hunk 5 above)
Comment (jcsuarezm) [reply service/src/main/resources/application.yaml]: Thank you Julián, this works for multiple topics, but to maintain uniformity I have adopted the pattern used in other services.

**Pull request merged**